### ISSUE TITLE
Birthday calendar

### ISSUE THREAD

**Issue by phsc84** [libasys/calendarplus#1]
It would be nice, if birthdays of contacts could be displayed as an own (read-only!) calendar. This calendar should also be accessible via CalDAV.
##

**Comment by phsc84** [comment]
I hope, it is ok, that I've opened a new issue in the new repo as a copy of the old issue.

**Comment by libasys** [comment]
yes of course, this comes with contactsplus app! than the birthdays of the contacts are in a seperate calendar!

**Comment by phsc84** [comment]
Great! :+1:

**Comment by sahne123** [comment]
You are working on a contactsplus app? Why that? I think i know the reasons for the calendar and tasks, but wouldn't it be reasonable to contribute on the official contacts app?

What are your plans with contactsplus?

**Comment by libasys** [comment]
It is easy to say i don't like the layout of the contacts app!

**Comment by sahne123** [comment]
I dont like it either. I'm just sceptical of building a complete new incompatible ecosystem to the official apps. I think we all agree that the offical calendar and contacts app has to be improved. Going forward, i hope some day there will be only one calendar/task/contacts app or at least fully compatible apps.

Don't get me wrong, ich love calendarplus for filling some of the biggest holes in calendar!

**Comment by libasys** [comment]
@sahne123 a beta of contactsplus i release today here on git for testing! And a migration script for existing calendars will be released in the near future! Migration should happend by one click! (Small migration button on the settings dialog of the calendar+ app!)

**Comment by sahne123** [comment]
I'm looking forward to this release and will be happy to test ist. Maybe your version will also fix my current problems with the standard contacts app (moving contacts from one addressbooks to antother, no search anymore) _fingers crossed_

**Comment by libasys** [comment]
yepp you can moveror copy contacts between addressbooks! Search is integrated! but today the release is no more possible! (offtopic)mini golf ist dazwischen gekommen(/offtopic)

**Comment by JonathanCrouse** [comment]
So I've been using this today and I do have the Birthday subscription calendar showing in the web interface. I take it that it is not yet available via CalDAV yet as I'm seeing it as an option.  Is this correct or am I missing something?

**Comment by Mr-HeBo** [comment]
I miss the birthdays of my contacts too :(. These were displayed in the standard calendar but not in calendar plus.

**Comment by phsc84** [comment]
As @libasys already commented, you have to use his Contacts+ app. Then the birthdays will be shown. At least it works perfectly for me.

**Comment by Mr-HeBo** [comment]
thanks :). It's true. But the birthday calendar will not synchronised with my mobile devices (iCal Import / DAVdroid).

**Comment by matzex** [comment]
Same problem for me. With the "standard" calendar app i can use the birthday calendar app in every client, which is currently not possible. Also it would be nice if the birthday calendar would work with the standard contact app.

**Comment by deMattin** [comment]
As @Mr-HeBo mentioned, the "virtual" birthday calendar is not syncable with caldav.
It is only shown in web app of calendarplus and it uses contactsplus for the birthday data.
It would be nice to integrate this generated calendar also in caldav as a readonly calendar like it is in "official" calendar.

**Comment by imago-maeror** [comment]
I agree with @deMattin and @Mr-HeBo
Same problem here, I try to sync the birthday calendar via caldav with Thunderbird/Lightning and with CalDAV-Sync (Android). I just hoped that there is a caldav link like "http(s)://DOMAIN/remote.php/caldav/calendars/USER/contact_birthdays" in the official calendar app, but I couldn't find any.

**Comment by libasys** [comment]
comes with a later version of calendar+! every user get his own birthday calendar if contacts+ app is enabled! and so the sync is no problem anymore!

**Comment by libasys** [comment]
birthday calendar is now available per caldav with contacts+ version greater 1.0.5
you have to use current master calendar+ and current master of contacts+
